Sponsors.md

```diff
@@ -1,90 +1,80 @@
 # Sponsors
 
-We thought long and hard about accepting sponsorships. You will find our thinking in the “History” section, at the end of this page. 
+## Main way to sponsor: OpenCollective
 
-_These days, it seems, everyone has some advertisement targeted to software developers..._
+The main way to sponsor DevStaff is through [our OpenCollective page](https://opencollective.com/devstaff), which helps us gather financial contributions via a transparent system. There, you can find the available contribution options such as:
 
-Below you will find the available sponsorship packages / options: 
+* One-time contribution of a custom amount
+* One-time contributions of predefined amounts to sponsor a number of next meetups
 
-## DevStaff Swag Sponsor
-Help the community members wear the DevStaff community name with pride! With this option, you will sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+As a thank you for sponsoring a number of the upcoming meetups, we will feature you in those meetups by mentioning you and your contribution.
```

Comment:
I think that it should be mentioned what other benefits sponsors get when sponsoring via open collective. For example, they are promoted via social media announcements prior to the meetup and also on post-meetup thank you messages.

They are also included (but we should decide on when) on [this](https://www.devstaff.gr/sponsors) page of our website.

Reply:
If I can recall correctly we only mention them, after the meetup.

Reply:
I suppose that the complete list is this:
```
* leaflets (can be left on a table at entrance),
* online banner on DevStaff website (with back-link do-follow),
* "Thank you" message by organizers during Meetup (not by company),
* Sponsor's slide in DevStaff Welcome presentation,
* "Thank you" mention in our social media.
```

Reply:
Thanks! I made a new section "Awareness benefits" that tries to gather the standard ways we promote a sponsor. Please review.

Reply:
Correct, social media include sponsors on post-meetup "thank you" posts.

Apart from the list that you mentioned @mzampetakis , I think that we also include the sponsors of the monthly meetup in meetup.com announcement (if we want to make it more explicit on what sponsors get as a benefit and also be more precise on what refers to the meetup time range and what is available online before and after the meetup).

So, as a conclusion, I would prefer to have a complete list of what we offer as a return of a sponsorship instead of just mentioning `we will feature you in those meetups by mentioning you and your contribution.`